package.json

```diff
@@ -19,7 +19,7 @@
     "precommit": "lint-staged",
     "prepublishOnly": "yarn build",
     "test-ci": "lerna run test-ci --stream",
-    "test": "lerna run test --stream"
+    "test": "mocha packages/**/*.test.js --require @parcel/babel-register"
```

Comment:
We've started using the extension for `.test.js` for new unit tests, but the integration tests do not have that extension yet. One option would be to change the glob to just look for any files inside of the `test` folder at the root of all packages, but this would also be a good opportunity to consistently name our tests with the `.test.js` extension.